informer/informer.go

```diff
@@ -286,7 +286,7 @@ func (i *Informer) newWatcher(ctx context.Context) (watch.Interface, error) {
 				return microerror.Mask(err)
 			case <-found:
 				// fall through
-			case <-time.After(time.Second):
+			case <-time.After(5 * time.Second):
```

Comment:
Here is another improvement I would like to go with. Not sure if 5 seconds as compared to the former one second is too much, but I think on certain installations we do not meet the one second criterion all the time and relaxing this could lead to a more reliable service quality. In cases where this is still not good enough, we have to get our hands on the root cause of the control plane APIs. 

I would like to have some proper alerting on error rates but @giantswarm/sig-monitoring has to provide some guidance here and so far we do not have a really cool solution for that.